executor/executor.cc

```diff
@@ -14,7 +14,18 @@
 #include <stdlib.h>
 #include <string.h>
 #include <time.h>
+
+#if GOOS_windows
```

Comment:
Please try to move all of this into executor_windows.h

Reply:
Moving all this to common_windows.h seems viable as it is already included in most of the places. But the only issue would be with position of [```csum_inet_update```](https://github.com/google/syzkaller/blob/cb436c69d9bcb0330518a48559649c9436ed5e7a/executor/common.h#L365) which comes above [```#include common_windows.h```](https://github.com/google/syzkaller/blob/cb436c69d9bcb0330518a48559649c9436ed5e7a/executor/common.h#L398).

Reply:
htobe* definitions need to go after the following block in common.h where we handle differences for these functions across other OSes. htobe* are also needed for C reproducer, so they need to be in common*.h.
```
#if GOOS_freebsd || GOOS_test && HOSTGOOS_freebsd
#include <sys/endian.h> // for htobe*.
#else
#include <endian.h> // for htobe*.
#endif
```

The remaining things can go into executor_windows.h, rigth?

Reply:
Okay yeah I will do the same and get all the Windows specific functions together in one place.